fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg

```diff
@@ -1,26 +1,6 @@
 ; Copyright (c) 2023-2024 FlyByWire Simulations
 ; SPDX-License-Identifier: GPL-3.0
 
-; A380 MSN 225 - WV03 A380-842
-; 07 Mar 2023 - Fuel System Update
-
-; 10 Feb 2024 - Masses (tanks/cargo) shift 5m fwd to match visual model
-; Contact point of wing gear slightgly set higher so it gets less plane weight (shift weight on nose)
-; Main gears contact points pushed a bit aft as a compromise between correct tilt animation and weight distribution
-
-; 14 Feb 2024 - Moved CG while keeping correct %Mac offset by moving aero center forward
-; Changed contact points and empty mass to match real MSN empty weight load sheet
-
-; 15 Feb 2024 - Activate new spring method and update strut springs
-
-; 19 Jul 2024 - Adjust CG, wheels and fuel tanks to correct model origin to Airbus reference datum offset
-; The position of the nose in the model coord. system is 35.1927m. The Airbus reference datum distance from the nose is 7.33m.
-; So, the conversion from HArm value to model coordinates in m is: model_coord = (35.1927m + 7.33m) - harm_value
-
-; 13 Sept 2024 fixed some invalid entries in the trigger section
-
-; 22 Oct 2024 Fixed numerous issues with the fuel system. Code added to address MSFS fuel system issues and restrict transfers until airplane is in flight.
```

Comment:
Was this meant to be deleted?

Reply:
Yes. All this was good for development, but I feel superfluous now. All future modifications will be documented in their respective PRs.